desk/set_up/get_inputs.py

```diff
@@ -0,0 +1,28 @@
+# Steve Goldman, Space Telescope Science Institute, sgoldman@stsci.edu
+__all__ = ["users"]
+
+
+class users(object):
+    """Initializes user inputs."""
+
+    def __init__(self, source, distance, grid):
+        """
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Multi-line docstring summary should start at the first line (D212)](https://app.codacy.com/manual/s-goldman/Dusty-Evolved-Star-Kit/pullRequest?prid=5418172)